### PR TITLE
Dashboard Linked from Login

### PR DESCRIPTION
**Pull request by tailaiwang**
Notice changed packages, see comments for missing details.

### CODE REVIEW

File: src/views/DashboardPage/index.js [hunk 1]
```diff
@@ -1 +1,74 @@
...
+      }
+      //Set the profile info
+      cognitoUser.getUserAttributes(function (err, result) {
+        if (err) {
+          return;
+        }
+      });
+    });
+  } else{
+    document.getElementByIdById("display_error").innerHTML = "Not Logged In";
```
Comment (dunja-tomic) [inline src/views/DashboardPage/index.js:32]: Redirect the user to /login if they aren't signed in

File: src/views/DashboardPage/index.js [hunk 2]
```diff
@@ -1 +1,74 @@
...
+};
+
+const userPool = new CognitoUserPool(poolData);
+var cognitoUser = userPool.getCurrentUser();
+
+window.onload = function () {
+  if (cognitoUser != null) {
+    cognitoUser.getSession(function (err, session) {
+      if (err) {
+        alert(err);
```
Comment (dunja-tomic) [inline src/views/DashboardPage/index.js:21]: Does this mean the user isn't logged in? In which case should they be redirected to /login?

File: package.json [hunk 3]
```diff
@@ -3,14 +3,16 @@
...
-    "chart.js": "^2.9.3",
-    "react-bootstrap": "^1.0.0",
-    "react-chartjs-2": "^2.9.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "amazon-cognito-identity-js": "^4.2.1",
+    "aws-sdk": "^2.656.0",
     "bootstrap": "^4.4.1",
+    "chart.js": "^2.9.3",
```
Comment (dunja-tomic) [inline package.json:12]: Do we need chart.js and react-chartjs-2?

Comment (tailaiwang) [reply package.json:12]: I was planning on using it for the Statistics section for dope circles, not needed rn though.

**Comment by tailaiwang** [comment]
Changes Made! <3

**Pull request merged**